website/source/docs/job-specification/update.html.md

```diff
@@ -79,8 +79,10 @@ job "docs" {
 
 - `healthy_deadline` `(string: "5m")` - Specifies the deadline in which the
   allocation must be marked as healthy after which the allocation is
-  automatically transitioned to unhealthy. This is specified using a label
-  suffix like "2m" or "1h".
+  automatically transitioned to unhealthy. This is specified using a label suffix
```

Comment:
This is a good improvement. 

I wonder if adding:

```
... automatically transitioned to unhealthy, after which it will be rescheduled according to the reschedule stanza (link) if the update is still within the `progress_deadline`. ...
```

Reply:
Good call. The reschedule stanza is really the missing link to how these 2 settings interact.